pkg/cmd/server/admin/create_signercert.go

```diff
@@ -99,9 +96,9 @@ func (o CreateSignerCertOptions) CreateSignerCert() (*crypto.CA, error) {
 		ca, written, err = crypto.EnsureCA(o.CertFile, o.KeyFile, o.SerialFile, o.Name)
 	}
 	if written {
-		fmt.Fprintf(o.Output.Get(), "Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
+		glog.V(3).Infof("Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
```

Comment:
We have log levels for this.

> On Jun 5, 2015, at 8:51 AM, Luke Meyer <notifications@github.com> wrote:
> 
> In pkg/cmd/server/admin/create_signercert.go:
> 
> > @@ -99,9 +96,9 @@ func (o CreateSignerCertOptions) CreateSignerCert() (*crypto.CA, error) {
> >  		ca, written, err = crypto.EnsureCA(o.CertFile, o.KeyFile, o.SerialFile, o.Name)
> >  	}
> >  	if written {
> > -		fmt.Fprintf(o.Output.Get(), "Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
> > +		glog.V(3).Infof("Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
> Because otherwise, when you run cert generation commands, you get no feedback on what it actually did. Also if you're running "openshift start" directly you have a 10-second wait with no feedback while it generates all keys/certs.
> 
> I'd have no issue with a flag to suppress these, or even suppressing them by default for "openshift start master" since you probably don't need to see this in the journal.
> 
> What's the objection to having this output?
> 
> —
> Reply to this email directly or view it on GitHub.
>